osu.Game/Screens/Edit/Compose/Components/EditorSelectionHandler.cs

```diff
@@ -281,6 +281,8 @@ private void resetTernaryStates()
             SelectionAdditionBanksEnabled.Value = true;
             SelectionBankStates[HIT_BANK_AUTO].Value = TernaryState.True;
             SelectionAdditionBankStates[HIT_BANK_AUTO].Value = TernaryState.True;
+            foreach (var (_, sampleState) in SelectionSampleStates)
```

Comment:
I'm not sure this is actually better behaviour than just leaving the last state, but let's go with the theme of matching stable until people complain.